src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -72,13 +72,20 @@ export default {
       type: Object,
       required: false,
     },
+    standardColorIdentifier: {
+      type: String,
+      required: false,
+    },
   },
   computed: {
     // get the alias, if any, and fallback to the `teal` color
     color: ({ type }) => HeroColorsMap[TopicTypeAliases[type] || type] || HeroColors.teal,
-    styles: ({ color }) => ({
+    styles: ({ color, standardColorIdentifier }) => ({
       // use the color or fallback to the gray secondary, if not defined.
-      '--accent-color': `var(--color-documentation-intro-accent, var(--color-type-icon-${color}, var(--color-figure-gray-secondary)))`,
+      '--accent-color': `var(--color-documentation-intro-accent, var(--color-type-icon-${color}))`,
```

Comment:
This is more of a design comment and not an issue with the code, but I think we should explore deriving the color of the icon in the hero based on the provided "standard" color (if any), because the static icon color really sticks out now when the background color changes.

Reply:
@SamLanier ^^

Reply:
Could be a future enhancement, but not tied to this specific PR.